webapp/src/ts/services/header-tabs.service.ts

```diff
@@ -76,4 +80,15 @@ export class HeaderTabsService {
 
     return this.tabs;
   }
+
+  canAccessTab(tabName: string): Promise<boolean> {
```

Comment:
Nice!

Reply:
This isn't needed anymore since we'll load the modules always, but I can take a look at the code maybe somewhere else can use it, otherwise I'll remove it 😅

Reply:
I've removed the function since it's not needed anymore